### PR TITLE
[BPK-1177] Adding horizontal nav to docs site header

### PR DESCRIPTION
**Pull request by jamesf3rguson**

### CODE REVIEW

**Comment by k0nserv** [comment]
This approach unfortunately does not work in Firefox and Edge. Even in Chrome and Safari where it does work you can only click the text rather than the whole item which is meh at best

File: packages/bpk-docs/src/components/Header/Header.js [hunk 1]
```diff
@@ -56,68 +56,90 @@ const toNavList = (links, hideOnTablet) => {
...
+    };
+    itemProps.className = getClassName(
+      'bpkdocs-header__horizontal-nav__nav-list-link',
+    );
+  } else {
+    itemProps.href = link.href;
+    itemProps.target = link.blank ? '_blank' : '_self';
+    itemProps.className = getClassName(
+      'bpkdocs-header__horizontal-nav__nav-list-link',
+    );
```
Comment (shaundon) [inline packages/bpk-docs/src/components/Header/Header.js]: (sorry for the repeat comment, I added it in the wrong place first)

This looks like it's repeated in both the `if` and `else`, can we move it up?

Comment (k0nserv) [reply packages/bpk-docs/src/components/Header/Header.js]: Yeah this code can definitely be refactored anyway, will do if we agree with the approach

Comment (shaundon) [reply packages/bpk-docs/src/components/Header/Header.js]: Ok cool - in that case I definitely agree with the approach :)

**Comment by k0nserv** [comment]
Updated :) @jamesf3rguson can you try this out and make sure you like the look/feel?

**Pull request merged**